src/main/content/_assets/css/home.scss

```diff
@@ -86,50 +78,27 @@ body{
     height:120px;
     background-color: #d8d8d8;
     color: #000000;
-    border:5px solid #000000;
     cursor: pointer;
 
     &:hover{
         background-color: #FFFFFF;
     }
-
-    &:focus{
```

Comment:
Any reason you went with the blue?  Now the selected box gets a blue outline instead of a Kabanero colored outline
<img width="233" alt="Screen Shot 2019-09-05 at 9 12 34 AM" src="https://user-images.githubusercontent.com/37549026/64344897-5b5b7600-cfbd-11e9-91c2-adc1532a9ed6.png">

<img width="214" alt="Screen Shot 2019-09-05 at 9 09 09 AM" src="https://user-images.githubusercontent.com/37549026/64344733-09b2eb80-cfbd-11e9-9745-7a0949bb200e.png">